src/main/java/seedu/duke/commands/EditPatientCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.duke.commands;
```

Comment:
Is this just editing patient tag only? If yes, need a better name or this to edit all edit patient parameter in the future?

Reply:
For now is just to edit tags. However in the addressbook3, I see that they have multiple parameters that we don't have yet. So maybe we need to discuss this further to come to a conclusion.